modules/secured-cluster-services-config.adoc

```diff
@@ -1,6 +1,10 @@
 // Module included in the following assemblies:
 //
-// * dir/filename.adoc
+// * installing/installing_helm/install-helm-customization.adoc
```

Comment:
I cannot verify "installing/installing_helm/install-helm-customization" or the cloud services include locations.

Reply:
@dfitzmau I ran the `list_assemblies_used_by_modules.sh` to update the assemblies list. I think the issue is that that assembly is not included in the topic map. I think we can merge this one, as it is and I'll create a docs Jira issue to remove unused assemblies from the docs.

Reply:
Hi @gaurav-nelson . Thanks for the update. Should installing_helm/install-helm-customization.adoc be completed removed from the lastest `rhacs` branch? Could this work be done in this PR?

Reply:
Would you be OK to remove `* installing/installing_helm/install-helm-customization.adoc` from this file if you plan on removing the assmbly anyway?

Reply:
Thank you @dfitzmau I've removed the file path from the module. I'll create a Jira issue to remove unused files.